README.md

```diff
@@ -38,6 +42,8 @@ There is a docker image `telefonica/prometheus-kafka-adapter:1.8.0` [available o
 
 Prometheus-kafka-adapter listens for metrics coming from Prometheus and sends them to Kafka. This behaviour can be configured with the following environment variables:
 
+- `PROM_METADATA_ENDPOINT`: defines prometheus metric metadata endpoint , not set by default and hence metadata wont be included.
```

Comment:
What if we just made this a boolean instead (e.g. `ENABLE_PROM_METADATA`)? Requiring the endpoint may be a bit confusing.